### ISSUE TITLE
Bug: initViewEngine should be removed

### ISSUE THREAD

**Issue by lirantal** [lirantal/Riess.js#7]
Since Riess.js is API-centric the view engine isn't relevant and should be removed from `express.js`

**Comment by mleanos** [comment]
Agreed. I'll look at this, as I start re-familiarizing myself with Riess.

Just to clarify, the idea is that we'll have a simple server (running independently of the server-side api) implementation for the client-side app, so we could have server-side rendered templates?

**Comment by lirantal** [comment]
Right. The server-side API is completely view-free.
If the client-side app doesn't need any SSR then it can be a just a frontend bundled assets hosted on some CDN and no actual server work is required to process requests. Otherwise, if SSR is needed then that's a different project than Riess's server-side API.
